webtask.json

```diff
@@ -27,6 +27,7 @@
     },
     "LOGSTASH_URL": {
       "description": "Logstash URL (from logstash-input-http)",
+      "example": "https://listener.logz.io:8071",
```

Comment:
why did you use this as the example URL? I thought `logz.io` was something different?

Reply:
I just copied that from my local config. I used to have free trial logstash instance in logz.io